common4j/src/main/com/microsoft/identity/common/java/cache/BrokerOAuth2TokenCache.java

```diff
@@ -809,7 +785,7 @@ public AccountRecord getAccount(@Nullable final String environment,
                 result = cacheIterator
                         .next()
                         .getAccount(
-                                environment,
+                                null,
```

Comment:
Did a quick double-take here... looks like you're just being explicit rather than relying on the `environment` being `null` as I had... LGTM